internal/pkg/errorconcept/handler.go

```diff
@@ -26,7 +26,7 @@ type ErrorConceptType interface {
 	message(err error) string
 }
 
-type handler struct {
+type Handler struct {
```

Comment:
Why was this made public?

Reply:
- That was done to export it. 
- Otherwise, the following code (in `/internal/core/metadata/container/error_handler.go`), which is under `metadata`, won't be able to access it, resulting in these compile-time errors:
```
internal/core/metadata/container/error_handler.go:23:46: cannot refer to unexported name errorconcept.handler
internal/core/metadata/container/error_handler.go:26:36: cannot refer to unexported name errorconcept.handler
internal/core/metadata/container/error_handler.go:27:33: cannot refer to unexported name errorconcept.handler
```

Reply:
Also running into the issue; commenting to see what the resultant solution will be.

If we don't want to export the default implementation, I supposed we could duplicate it per service?

Reply:
@brandonforster Yep, let's stay in touch regarding this so we can have an aligned, uniform solution  👍

Reply:
Change to make `Handler` public is appropriate.  Since both you and Brandon need what you've got in .../container/error_handler.go, consider a common implementation in internal/pkg/container that both of you use (first merge wins).  

@akramtexas @brandonforster

Reply:
@michaelestrin Excellent, thanks for the feedback and confirmation 👍